samples/scala-protobuf-tracing/.env

```diff
@@ -0,0 +1,2 @@
+USER_SERVICE_HOST=host.docker.internal
+USER_SERVICE_PORT=8080
```

Comment:
Same as in the other PR. We need to align the files.

Reply:
done